src/pages/Popup/index.jsx

```diff
@@ -1,6 +1,5 @@
 import React from 'react';
 import { render } from 'react-dom';
-
 import Popup from './Popup';
```

Comment:
It's a common way to seperate the inner modules and 3rd modules by a blank line. And it's also a practice to place the inner modules on the top of the file, then following with the 3rd modules (so errors of inner modules can be exposed faster).

Reply:
Received ~ This is a better coding style.